crates/volta-core/src/tool/package/mod.rs

```diff
@@ -137,10 +137,12 @@ impl Display for Package {
 /// * the shims
 /// * the unpacked and initialized package
 pub fn uninstall(name: &str) -> Fallible<()> {
+    let mut package_found = false;
```

Comment:
I think we can actually avoid a mutable boolean value and instead assign the value of the if statement directly. That is, we can do something like:

```rust
let package_found = if package_config_file.exists() {
    ...
    true
} else {
    let package_binary_list = binaries_from_package(name)?;
    if !package_binary_list.is_empty() {
        ...
        true
    } else {
        false
    }
};
```

And then follow it up with the `if package_found` check below to determine what output to write.

Reply:
cool love that 👍